include/cute/atom/copy_traits_xe.hpp

```diff
@@ -505,12 +505,12 @@ CUTE_HOST_DEVICE constexpr auto make_fragment_layout(TiledCopy &tiled_copy,
       Int<!TiledCopy::is_convention_MN ? size<1>(mma_atom_shape_2d) : size<0>(mma_atom_shape_2d)>{};
   Int copy_size_M =
       Int<!TiledCopy::is_convention_MN
-              ? size<1>(typename TiledCopy::BlockShape{}) /
+              ? size<0>(typename TiledCopy::BlockShape{}) /
```

Comment:
you could shorten the code by making just the second argument to division a ternary expression

Reply:
Tidied it up a bit :+1: thanks